src/main/java/Duke.java

```diff
@@ -1,10 +1,181 @@
-public class Duke {
+import duke.Storage;
+import duke.TaskList;
+import duke.Ui;
+import duke.DukeException;
+import javafx.application.Application;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.Label;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import javafx.stage.Stage;
+import javafx.scene.layout.Region;
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+
+/**
+ * CS2103T AY 19/20 S2
+ * Individual Project
+ * @author Yap Dian Hao
+ * @since 2020-01-30
+ * <p>
+ * Duke is a personal chatbot assistant to help you organize your tasks. Especially helpful
+ * for people who love to procrastinate. Or procrastinate to procrastinate.
+ * </p>
+ */
+
+public class Duke extends Application {
+
+    private Image user = new Image(this.getClass().getResourceAsStream("/cuteuser.png"));
+    private Image duke = new Image(this.getClass().getResourceAsStream("/cutebot!.png"));
+    private ScrollPane scrollPane;
+    private VBox dialogContainer;
+    private TextField userInput;
+    private Button sendButton;
+    private Scene scene;
+    private Storage storage;
+    private TaskList taskList;
+    private Ui ui;
+    private String filepath;
+
+
+
+    /**
+     * Constructor of the Duke chatbot assistant.
+     * Duke is able to perform CRUD operations (Create, Read, Update, Destroy) to manage tasks.
+     * @param filePath A text file which functions as a database to store all tasks.
+     */
+
+    public Duke() {
+        this.filepath = "data/duke.txt";
```

Comment:
Could try declaring filepath as a static final string to combine the two constructors